README.org

```diff
@@ -1,17 +1,46 @@
 #+TITLE: Ren'Py mode
 
-PyTom's old major mode for Ren'Py, previously maintained by treymerkley, now maintained by me.
+A major mode for editing Ren'Py files.
 
 ** Installation
 
   Either install within Emacs directly from MELPA or:
+
   * Download/clone this repo to your desired location.
+
   * In Emacs, use the command =M-x package-install-file= and point it to the renpy.el file location
```

Comment:
Technically `M-x package-install-file` is not a command, `package-install-file` is the command and `M-x` is the keybinding to use to run a command that prompts for a command to run.